spec.bs

```diff
@@ -947,11 +959,11 @@ A <dfn for=fencedframetype>destination event</dfn> is either a
        beacons](https://wicg.github.io/attribution-reporting-api/#ref-for-request-service-workers-mode).
 
     : [=request/origin=]
-    :: <span class=XXX>Get the origin from somewhere, like the [implementation
-       does](https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:content/browser/fenced_frame/fenced_frame_reporter.cc;l=384;drc=183dd920ea5d6aed92e007c40e226f1f840c0567)</span>.
+    :: |request initiator| if |event| is a [=fencedframetype/destination URL event=],
```

Comment:
It's not obvious to me why it's important (or even safe) to change the request's origin based on this criteria. Can you add a note explaining why?

Reply:
I'll add a note to this.

[See this part of the design doc for a more detailed explanation](https://docs.google.com/document/d/1gKRZ9g_X5HCZbW__GinViwcC8iWEai_kBXsiJCXrt80/edit#bookmark=id.y3ru3k6q9pvv).